packages/expo-permissions/ios/EXPermissions/EXPermissionRequester.h

```diff
@@ -3,16 +3,16 @@
 #import <Foundation/Foundation.h>
 #import <UMCore/UMExportedModule.h>
 
+@class EXPermissions; // forward declaration 
+
 @protocol EXPermissionRequester <NSObject>
 
-- (void)setDelegate:(id)permissionRequesterDelegate;
-- (void)requestPermissionsWithResolver:(UMPromiseResolveBlock)resolve
-                              rejecter:(UMPromiseRejectBlock)reject;
+- (instancetype)initWithPermissionsModule:(EXPermissions *)permissionsModule;
 
-@end
+- (void)requestPermissionsWithResolver:(UMPromiseResolveBlock)resolve
 
-@protocol EXPermissionRequesterDelegate <NSObject>
+                              rejecter:(UMPromiseRejectBlock)reject;
 
-- (void)permissionRequesterDidFinish:(NSObject<EXPermissionRequester> *)requester;
+- (NSDictionary *)permissions;
```

Comment:
I definitely would like to have more descriptive method name 🤔 Taking a look at `permissions` method name gives me absolutely no clue what this method is about.
Good way for naming your methods it to use at least one verb and most probably we would like to have sth like `getPermissions` or maybe something connected with `check` or other work that better suits this context 🤔 what do you think?